docs/guide/pieces.md

```diff
@@ -1,396 +1,85 @@
----
-title: "Pieces"
----
-
 # Pieces
-What is a Piece?
 
-As experienced A2 developers know, a piece type module defines structure and behavior for a single type of content, such as blog posts, products, or events. If you have hundreds of documents of the same type, that's a piece. If you want to let the user view them in a certain sorting order, that's a piece. If the same item might be appropriate to display on many different pages, that's a piece. Blog posts, events and products are all excellent examples. Less obvious examples include "category" or "tag" content types that serve to create a taxonomy for other pieces.
+"Piece types" are classifications for standalone, structured content. Each database document created with a piece type module is known as a "piece". The piece type defines how pieces are structured, how end users interact with them, and how they might interact with other content. Blog posts, events, and people profiles are all common examples of piece types.
+
+Use cases for pieces include:
+- Content that will be listed using pagination, such as **articles in a blog**
+- Content of a particular type that will help to organize other content, such as **teams that connect to people pieces**
```

Comment:
This one is confusing. Can we use something like "The **author** of an article".